### ISSUE TITLE
Support HTML event descriptions

### ISSUE THREAD

**Issue by RockyTV** [Etar-Group/Etar-Calendar#308]
I am synced to a calendar that uses HTML for its description (it's like a daily menu calendar for a restaurant and they use HTML). Google Calendar renders it perfectly, while AOSP and Etar do not.

**Comment by dilyanpalauzov** [comment]
How does the iCalendar look like?  Is it an [ALTREP](https://tools.ietf.org/html/rfc5545#section-3.2.1) property parameter?

**Comment by RockyTV** [comment]
No, it was straight into the description field. What surprised me is that Google Calendar rendered it fine, the other apps wouldn't render the HTML. However, from what I understood from the RFC, you can't use HTML in the description field.

**Comment by dilyanpalauzov** [comment]
So, how do you want to have this implemented?

**Comment by MartinX3** [comment]
Same issue here.
I use the google calendar.

The browser calendar renders it correctly.
The thunderbird lightning addon renders it correctly.
The etar calendar app shows html code.

**Comment by drdimi** [comment]
Any update on this would be great if supported

**Comment by seindal** [comment]
A Google calendar from people I work with does this.
There are so many tags it's hard to read the text.

For me it would be sufficient just to strip the tags.

**Comment by skwde** [comment]
Nowadays many calendar provider (google / outlook / ...) which are used by companies seem to use HTML.

Is support for this finally on the roadmap after almost 6 years?

**Comment by SimonLammer** [comment]
Is there any update to this. Would love myself some readable calendar entry descriptions...
